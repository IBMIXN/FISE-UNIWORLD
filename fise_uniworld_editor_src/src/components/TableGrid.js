import PropTypes from 'prop-types';
import { MeetingTablePropType } from '../propTypes/EventRoom';
import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import { Avatar } from '@material-ui/core';
import { isEdge, getTableIndex, getTableImage } from '../utils/tableUtils';

const useStyles = makeStyles((theme) => ({
  gridContainer: {
    display: 'grid',
    gridGap: '5px',
    gridTemplateColumns: 'repeat(9, 50px)',
    gridTemplateRows: 'repeat(9, 50px)',
    [theme.breakpoints.down('xs')]: {
      gridTemplateColumns: 'repeat(9, 36px)',
      gridTemplateRows: 'repeat(9, 36px)',
    },
    gridAutoFlow: 'row',
  },
  gridItem: {
    display: 'flex',
    textAlign: 'center',
    alignItems: 'center',
    justifyContent: 'center',
  },
  gridItemFill: {
    transition: 'background 0.3s ease',
    '&:hover': {
      background: '#aaa',
    },
    background: '#ccc',
  },
}));

const TableGrid = ({ tables }) => {
  const classes = useStyles();

  const generateGridItems = () => {
    const gridItems = [];
    for (let row = 0; row < 9; row++) {
      for (let col = 0; col < 9; col++) {
        const tableIndex = getTableIndex(row, col, tables);
        if (isEdge(row, col)) {
          gridItems.push(<div className={classes.gridItem}> </div>);
        } else if (tableIndex !== -1) {
          gridItems.push(
            <div
              id={tableIndex}
              className={`${classes.gridItem} ${classes.gridItemFill}`}
            >
              <Avatar src={getTableImage(tables[tableIndex].type)} />
            </div>
          );
        } else {
          gridItems.push(
            <div className={`${classes.gridItem} ${classes.gridItemFill}`}>
               
            </div>
          );
        }
      }
    }
    return gridItems;
  };

  return (
    <React.Fragment>
      <div className={classes.gridContainer}>
        {generateGridItems().map((gridItem, i) => (
          <React.Fragment key={i}>{gridItem}</React.Fragment>
        ))}
      </div>
    </React.Fragment>
  );
};

TableGrid.propTypes = {
  tables: PropTypes.arrayOf(MeetingTablePropType).isRequired,
};

export default TableGrid;
