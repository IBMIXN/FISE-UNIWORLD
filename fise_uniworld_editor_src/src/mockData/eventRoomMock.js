const mockEventRoom = {
  title: "mockTitle",
  eventDate: "2021-03-17T00:00:00.000Z",
  meetingTables: [],
  background: "mockBackground",
  scene: "mockScene",
};

const mockEventRoomUpdate = {
  title: "newMockTitle",
  eventDate: "2021-02-27T12:00:00.000Z",
  meetingTables: [],
  background: "newMockBackground",
  scene: "newMockScene",
};

const mockMeetingTable = {
  title: "mockTitle",
  type: "RoundMeetingTable",
  logoUrl: "https://via.placeholder.com/150",
  zoomUrl: "https://zoom.us/",
  posX: 0,
  posY: 0,
};

export { mockEventRoom, mockEventRoomUpdate, mockMeetingTable };
