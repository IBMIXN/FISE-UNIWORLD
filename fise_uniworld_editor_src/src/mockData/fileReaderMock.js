export default class FileReaderMock {
  onerror = jest.fn();
  onload = jest.fn();
  addEventListener = (_, fn) => fn({ target: { result: {} } });
  readAsDataURL = jest.fn();
}
