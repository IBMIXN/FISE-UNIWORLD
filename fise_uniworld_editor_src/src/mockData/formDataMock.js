export default class FormDataMock {
  append = jest.fn();
}
