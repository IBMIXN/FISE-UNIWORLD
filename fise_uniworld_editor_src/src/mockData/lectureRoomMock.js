const mockLectureRoom = {
  title: "mockTitle",
  module: "mockModule",
  lecturer: "mockLecturer",
  startTime: "2021-03-17T09:00:00.000Z",
  endTime: "2021-03-17T10:00:00.000Z",
  firstSlideUrl: "mockUrl",
  numSlides: 1,
};

const mockLectureRoomUpdate = {
  title: "newMockTitle",
  module: "newMockModule",
  lecturer: "newMockLecturer",
  startTime: "2021-02-27T14:00:00.000Z",
  endTime: "2021-02-27T15:00:00.000Z",
  firstSlideUrl: "newMockUrl",
  numSlides: 3,
};

export { mockLectureRoom, mockLectureRoomUpdate };
