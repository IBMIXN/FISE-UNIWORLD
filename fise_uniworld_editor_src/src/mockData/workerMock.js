// https://github.com/webpack-contrib/worker-loader/issues/10#issuecomment-325977537
module.exports = Object.create(null);
