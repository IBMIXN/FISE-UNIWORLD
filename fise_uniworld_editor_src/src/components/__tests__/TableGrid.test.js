import React from 'react';
import { shallow } from 'enzyme';
import TableGrid from '../TableGrid';
import { mockMeetingTable } from '../../mockData/eventRoomMock';

describe('TableGrid test', () => {
  it('TableGrid should match snapshot', () => {
    const component = shallow(<TableGrid tables={[mockMeetingTable]} />);
    expect(component).toMatchSnapshot();
  });

  it('TableGrid should match snapshot with empty tables array', () => {
    const component = shallow(<TableGrid tables={[]} />);
    expect(component).toMatchSnapshot();
  });
});
