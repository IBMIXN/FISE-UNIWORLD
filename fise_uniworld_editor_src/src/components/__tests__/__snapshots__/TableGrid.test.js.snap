// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableGrid test TableGrid should match snapshot 1`] = `
<Fragment>
  <div
    className="makeStyles-gridContainer-1"
  >
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      id={0}
    >
      <WithStyles(ForwardRef(Avatar))
        src="roundtable.png"
      />
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
  </div>
</Fragment>
`;

exports[`TableGrid test TableGrid should match snapshot with empty tables array 1`] = `
<Fragment>
  <div
    className="makeStyles-gridContainer-1"
  >
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
  </div>
</Fragment>
`;
